test/SILOptimizer/definite_init_tuple.sil

```diff
@@ -0,0 +1,27 @@
+// RUN: %target-sil-opt -enable-sil-verify-all %s -definite-init -raw-sil-inst-lowering
+
+import Builtin
+import Swift
+import SwiftShims
+
+struct S<T> {
+  var x: (T, T)
+}
+
+sil [ossa] @$s19definite_init_tuple1SV1xx_xtvpfi : $@convention(thin) <T> () -> (@out T, @out T)
+
+sil [ossa] @$s19definite_init_tuple1SVACyxGycfC : $@convention(method) <T> (@thin S<T>.Type) -> @out S<T> {
+bb0(%0 : $*S<T>, %1 : $@thin S<T>.Type):
+  %2 = alloc_box $<τ_0_0> { var S<τ_0_0> } <T>, var, name "self"
+  %3 = mark_uninitialized [rootself] %2 : $<τ_0_0> { var S<τ_0_0> } <T>
+  %4 = project_box %3 : $<τ_0_0> { var S<τ_0_0> } <T>, 0
+  %5 = struct_element_addr %4 : $*S<T>, #S.x
```

Comment:
@slavapestov This has far fewer tests than the old version that was deleted above. Are you sure we didn't lose any code coverage?